sdk/monitor/monitor-opentelemetry-exporter/src/export/base.ts

```diff
@@ -28,7 +28,7 @@ export abstract class AzureMonitorBaseExporter {
   private readonly _sender: Sender;
   private _numConsecutiveRedirects: number;
   private _retryTimer: NodeJS.Timer | null;
-  private _statsbeatMetrics: StatsbeatMetrics | undefined;
+  protected _statsbeatMetrics: StatsbeatMetrics | undefined;
```

Comment:
protected is causing api check for StatsbeatMetrics class to be exported in index, not seeing this one being used outside of this class, maybe change it back to private?

Reply:
Looks like this was a holdover from a previous way I was using the _statsbeatMetrics. In any case, not necessary now, so I'll give that a change.